coqui-tts-gui.py

```diff
@@ -0,0 +1,202 @@
+# GUI solution for Coqui TTS made by AceOfSpadesProduc100.
+# WARNING: DO NOT run this from IDLE or double-clicking, it will be stuck loading. To be safe, you should enter "python mozilla-tts-gui.py" in your terminal such as command prompt or Powershell.
+# Make sure to edit this script to add TTS models and vocoders on the ['values'] of ttsmodelbox and vocodermodelbox.
+
+import tkinter as tk
+from tkinter import ttk
+from tkinter import scrolledtext
+from tkinter import filedialog
+from tkinter import messagebox
+import subprocess
+import sys
+import playsound
+from playsound import playsound
+from TTS.utils.synthesizer import Synthesizer
+from TTS.utils.manage import ModelManager
+import pathlib
+from pathlib import Path, PurePath
+import os
+
+def generate():
+    if inputbox.get("1.0","end-1c") == "":
+        messagebox.showerror(message = "TTS will give a division by zero error if the text field is blank.")
+    else:
+        if not os.path.exists('coqui-tts-output'):
+            try:
+                os.makedirs('coqui-tts-output')
+            except OSError as e:
+                if e.errno != errno.EEXIST:
```

Comment:
I guess it should be ```e.errno.EEXIST```

Reply:
this needs `import errno` for it to work